lib/pluginmanager/update.rb

```diff
@@ -34,14 +37,15 @@ def update_gems!
     # remove any version constrain from the Gemfile so the plugin(s) can be updated to latest version
```

Comment:
If you do `bin/plugin install --version '1.2' logstash-input-lumberjack` if will use a version in the gemfile, to update this plugin we remove the version and update to the latest.